packages/serverless/src/domain/use-case/rollup.ts

```diff
@@ -73,10 +74,10 @@ export async function createRollUp(
       previousMerkleTreeRoot: latestProofForDb.prevMerkleRoot,
       currentMerkleTreeRoot: latestProofForDb.merkleRoot,
       databaseName: databaseName,
-      transactionObjectId: txId,
+      transactionObjectId,
       proofObjectId: latestProofForDb._id,
-      status: ETransactionStatus.Unsigned,
-      transactionHash: '',
+      createdAt: new Date(),
```

Comment:
Use `getCurrentTime()` to use the cache time prevent to create multiple instance of `Date`

Reply:
done